tests/tests/legacy_kustomizations/knative-install/test_data/expected/apps_v1_deployment_autoscaler.yaml

```diff
@@ -29,8 +29,6 @@ spec:
     metadata:
       annotations:
         cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
-        sidecar.istio.io/inject: "false"
```

Comment:
Knative's recommendation is to disable sidecar in the control plane, are we disabling somewhere else now?

Reply:
None of the pods in the `knative-serving` namespace will have sidecars as I removed the `istio-injection: enabled` label from the `knative-serving` namespace that was originally there ([see change](https://github.com/kubeflow/manifests/pull/1617/files#diff-1400763df20c414e1df6ae00e938199f71c2cb62107262beed3076f4442ffc31)). The original Knative install YAMLs have istio-injection enabled, but this was causing issues as mentioned [here](https://github.com/kubeflow/manifests/pull/1617#issuecomment-723368026).

Reply:
Thanks @pvaneck ! we are good then.